src/v/config/configuration.cc

```diff
@@ -1662,7 +1662,34 @@ configuration::configuration()
       "balancer, in milliseconds",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       5000ms,
-      {.min = 1ms, .max = bottomless_token_bucket::max_width}) {}
+      {.min = 1ms, .max = bottomless_token_bucket::max_width})
+  , kafka_quota_balancer_node_period(
+      *this,
+      "kafka_quota_balancer_node_period_ms",
+      "Intra-node throughput quota balancer invocation period, in "
+      "milliseconds. Value of 0 disables the balancer and makes all the "
+      "thoughput quotas immutable.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      750ms,
+      {.min = 0ms})
+  , kafka_quota_balancer_min_shard_thoughput_ratio(
+      *this,
+      "kafka_quota_balancer_min_shard_thoughput_ratio",
+      "The lowest value of the throughput quota a shard can get in the process "
+      "of quota balancing, expressed as a ratio of default shard quota. "
+      "0 means there is no minimum, 1 means no quota can be taken away "
+      "by the balancer.",
```

Comment:
I actually think this setting is more useful than the absolute bps value, because it basically says, e.g. "5% of the shard's equal share of the node wide quota is always assigned to each shard no matter what". It's easier to reason about than the absolute value: we can see then that 95% can be taken away, which means that in an extreme case one shard can get up to 95% of the node wide limit.

But ultimately it's about convenience: since the strongest of the ratio and the absolute value is applied, it's up to the user which one to use.

Reply:
thanks!

since this doesn't fit into the configuration description, please add it to the commit message. you can write as much as you need to in the commit message to document stuff like this.